docs/Xamarin.Forms/Label.xml

```diff
@@ -60,7 +60,7 @@ public class App : Application
     };
 }
 ]]></code>
-        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://developer.xamarin.com/samples/tag/Xamarin.Forms/" target="_blank">Sample Applications</a></format> page, has a LabelDemoPage.cs file. This file contains a longer and more complete example.</para>
+        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/samples/" target="_blank">Sample Applications</a></format> page, has a LabelDemoPage.cs file. This file contains a longer and more complete example.</para>
```

Comment:
As before - these links point to both the specific example, and the more general page with all the XF samples :)
```suggestion
        <para>The <format type="text/html"><a href="https://docs.microsoft.com/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery sample</a></format>, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/samples/browse/?products=xamarin&term=Xamarin.Forms&terms=Xamarin.Forms" target="_blank">Sample Applications</a></format> page, has a LabelDemoPage.cs file. This file contains a longer and more complete example.</para>
```